share/spice/untappd/untappd.js

```diff
@@ -0,0 +1,71 @@
+(function (env) {
+    "use strict";
+
+    var normalize = function(item) {
+        var m = {
+            title: item.beer.beer_name,
+            subtitle: item.brewery.brewery_name,
+            url: "https://untappd.com/b/" +
+                    encodeURI(item.beer.beer_slug) + "/" +
+                    encodeURI(item.beer.bid),
+            icon: item.beer.beer_label,
+            style: item.beer.beer_style,
+            abv: item.beer.beer_abv,
+            ibu: item.beer.beer_ibu,
+            ratings: item.checkin_count,
+            created_at: item.beer.created_at,
+            content: item.beer.beer_description
+        };
+
+        if (!m.ibu) {
+            m.ibu = "No";
+        }
+
+        if (m.ratings >= 10000) {
```

Comment:
We actually have JS helper for this 👍 

https://github.com/duckduckgo/duckduckgo-utils/blob/master/util.js#L24